packages/terra-table/src/subcomponents/Cell.jsx

```diff
@@ -266,7 +266,13 @@ function Cell(props) {
         case KeyCode.KEY_RETURN:
           // Lock focus into component
           if (isGridContext && hasFocusableElements()) {
```

Comment:
It would probably be better to create a method that checks for elements that are not input based fields.  You need a way to keep your conditions in sync.  Doing it this way won't allow that to happen.

Reply:
Added.